resources/metadata/nexusUpload.yaml

```diff
@@ -0,0 +1,79 @@
+metadata:
+  name: nexusUpload
+  description: Upload artifacts to Nexus
+  longDescription: |
+    Upload build artifacts to a Nexus Repository Manager
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
```

Comment:
should be prefixed with `nexus`, see https://github.com/SAP/jenkins-library/blob/master/.github/CONTRIBUTING.md#jenkins-credential-handling